src/lightning_lite/cli.py

```diff
@@ -0,0 +1,177 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import os
+from argparse import ArgumentParser, Namespace
+from typing import List, Tuple
+
+from lightning_lite.accelerators import CPUAccelerator, CUDAAccelerator, MPSAccelerator
+from lightning_lite.utilities.device_parser import _parse_gpu_ids
+from lightning_lite.utilities.imports import _IS_WINDOWS, _TORCH_GREATER_EQUAL_1_13
+
+# torchrun in PyTorch 1.13.0 has a bug on the Windows platform and is thus not importable:
+# https://github.com/pytorch/pytorch/issues/85427
+if not (_IS_WINDOWS and _TORCH_GREATER_EQUAL_1_13):
+    import torch.distributed.run as torchrun
+else:
+    torchrun = None  # type: ignore[assignment]
```

Comment:
Can you make this a local import in `_torchrun_launch` so we don't need to check the `if` again? Also avoids setting a `None` default

Reply:
It would make the mocking of torchrun in tests incredibly hard. Shouldn't we be able to remove this code again once pytorch fixes the issue in one of their 1.13.x release? https://github.com/pytorch/pytorch/pull/88250

Reply:
I'm struggling to refactor all tests for the local import

Reply:
Then put the import and error into an utility fn that returns torchrun and mock this fn

Reply:
It eventually worked with monkeypatch. I hope it's ok now